apps/client/src/features/control/message/MessageControlExport.tsx

```diff
@@ -14,13 +15,14 @@ import style from '../../../views/editor/Editor.module.scss';
 export default memo(MessageControlExport);
 function MessageControlExport() {
   const isExtracted = window.location.pathname.includes('/messagecontrol');
+  const isLocked = getIsNavigationLocked();
   const classes = cx([style.content, style.contentColumnLayout]);
 
   return (
     <ProtectRoute permission='editor'>
       <div className={style.messages} data-testid='panel-messages-control'>
         {!isExtracted && <Corner onClick={(event) => handleLinks('messagecontrol', event)} />}
-        {isExtracted && <ViewNavigationMenu suppressSettings />}
+        {(isExtracted || isLocked) && <ViewNavigationMenu suppressSettings={isExtracted} isNavigationLocked={isLocked} />}
```

Comment:
question: what is the use case for this element to be locked and not extracted? would that not mean we need to have locked the editor which is not possible?

 I wonder if we can simply say (same for all the others)


```suggestion
        {isExtracted && <ViewNavigationMenu suppressSettings isNavigationLocked={isLocked} />}
```

or even inline the check for `isLocked` from the component body to avoid computing it if we are not extracted, like:

```suggestion
        {isExtracted && <ViewNavigationMenu suppressSettings isNavigationLocked={getIsNavigationLocked()} />}
```

Reply:
Changes made :)